qt-models/divepicturemodel.cpp

```diff
@@ -59,7 +59,7 @@ void DivePictureModel::updateDivePictures()
 		if (dive->selected) {
 			int first = pictures.count();
 			FOR_EACH_PICTURE(dive)
-				pictures.push_back({ dive->id, picture, picture->filename, {}, picture->offset.seconds });
+				pictures.push_back({ dive->id, picture, picture->filename, {}, picture->offset.seconds, {.seconds = 0}});
```

Comment:
Interesting ... I thought this will only be valid C++ starting with C++20...?

Reply:
Well, Travis likes it, so this is good to go for me (except the typo in CHANGELOG).